tests/logs/push/test_ami_push/test_aws_publish_failure_retry.txt

```diff
@@ -2,11 +2,13 @@
 [   DEBUG] Searching for product RHEL_HOURLY for provider awstest in rhsm
 [   DEBUG] Fetching product from https://example.com/v1/internal/cloud_access_providers/amazon/provider_image_groups
 [   DEBUG] 1 Products(AWS provider) in rhsm: RHEL_HOURLY(awstest)
+[    INFO] Upload image to AWS: started
 [    INFO] Uploading /tmp/aws_staged/region-1-hourly/AWS_IMAGES/ami-1.raw to region region-1 (type: hourly, ship: True)
 [    INFO] Image name: RHEL-8.5-RHEL-8.5.0_HVM_BETA-20210902-x86_64-5-Hourly2-GP2
 [   DEBUG] {"accounts": ["secret-1"], "arch": "x86_64", "billing_products": ["code-0001"], "container": "redhat-cloudimg-region-1", "description": "Provided by Red Hat, Inc.", "ena_support": true, "image_name": "RHEL-8.5-RHEL-8.5.0_HVM_BETA-20210902-x86_64-5-Hourly2-GP2", "image_path": "/tmp/aws_staged/region-1-hourly/AWS_IMAGES/ami-1.raw", "root_device_name": "/dev/sda1", "snapshot_account_ids": ["0987654321", "1234567890", "684062674729"], "snapshot_name": "RHEL-8.5-RHEL-8.5.0_HVM_BETA-20210902-x86_64-5-Hourly2-GP2", "sriov_net_support": "simple", "uefi_support": false, "virt_type": "hvm", "volume_type": "gp2"}
-[ WARNING] Unable to publish
-[    INFO] Retrying upload
+[   ERROR] Upload image to AWS: failed
+[ WARNING] Upload failed
```

Comment:
Just noting that I believe the test only showing "Upload failed", and no exception details, is due to how the test is written or how the caplog fixture works.  When running for real this should automatically show some exception details.